pkg/pillar/types/blob.go

```diff
@@ -71,7 +77,7 @@ func (status BlobStatus) LogCreate(logBase *base.LogObject) {
 	logObject.CloneAndAddField("state", status.State.String()).
 		AddField("datastoreid-uuid", status.DatastoreID).
 		AddField("size-int64", status.Size).
-		AddField("blobtype-int64", status.BlobType).
+		AddField("blobtype-int64", status.MediaType).
```

Comment:
MediaType is a string not an int64. Thus you should use a new field e.g., "media-type" in the log.

Reply:
Oops, nice catch. Fixing now.